### PR TITLE
Adds footer with advanced links

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_book: Ready to go!  Hits two issues - adds a footer, and puts advanced links on the page.

### CODE REVIEW

**Comment by bigtiger** [comment]
:+1:

File: css/style.css [hunk 1]
```diff
@@ -262,14 +262,10 @@ button.play-button.pause {
...
   padding: 1rem;
-}
-
-.how-to-remember,
-.definition,
-.how-to-contribute {
   -ms-flex-pack: justify;
   -webkit-box-pack: justify;
   -webkit-justify-content: space-between;
+
```
Comment (rylnd) [inline css/style.css]: Unnecessary line break here.

File: css/style.css [hunk 2]
```diff
@@ -308,7 +305,20 @@ h4 {
...
 
 .site-footer {
-  margin: 2rem 0;
+  border: 0 solid #ccc;
+  margin: 0 auto;
+  max-width: 900px;
+  padding: 1rem;
+  width: calc(100% - .5rem);
+}
+ .site-footer ul {
```
Comment (rylnd) [inline css/style.css:314]: You could put that extra newline down here, though :wink:

**Comment by rylnd** [comment]
At a high level, putting all these external links (and only external links) in the footer seems like the wrong place for them. I would expect the footer to have information about those that built the site, and maybe a sitemap if there's room to fill.

I'd expect these links to be on a separate 'External Resources' page (which is itself liked in the footer), but that's just spitballing.

The `<ul>` needs to be styled a bit. Maybe just a `list-style-type: none`.

I'll defer to the creative team on this, but I feel that the footer should be full-width, just like the header, not margin'd like the main content.

**Comment by kathyoleary** [comment]
Added corrections.  Discussed with client the suggestions on the footer and staying with the current setup with the option of changing in the future.  (fyi:  original footer did have info on  who set up the site and whatnot and chose to change it away from that)  Adding the advanced links to the footer as opposed to a new page also keeps this down to a single page format which has been what we have been consistently staying with to date.

**Pull request merged**